thrift/server.cc

```diff
@@ -41,7 +41,6 @@
 #include <limits>
 #include <cctype>
 #include <vector>
-#include <boost/make_shared.hpp>
```

Comment:
You can't just remove this or the build fails on old systems:

thrift/server.cc:99:34: error: ‘make_shared’ is not a member of ‘thrift_std’

You can wrap it with  "#ifdef THRIFT_USES_BOOST/#endif"

Reply:
Yep that's what I was testing indeed, fixed!